javascript-light-bulb/main.js

```diff
@@ -0,0 +1,14 @@
+var $circleButton = document.querySelector('.circle');
+$circleButton.addEventListener('click', switchColor);
+var $bodyBackground = document.querySelector('body');
+
+function switchColor(event) {
+  var currentColor = $circleButton.className.split(' ')[1];
```

Comment:
This is making your JavaScript dependent on data from the DOm to know what to do. As we discussed in our `dom-manipulation` Q&A, that is not ideal. Instead, you should make a variable in your javascript that represents whether the light is currently on or not, and give it a boolean value.